src/build/app/settings.rs

```diff
@@ -422,52 +446,32 @@ pub enum AppSettings {
     /// e.g. to test the applet without installing it
     /// or there may already be a command of that name installed.
     ///
+    /// To make an applet usable as both a multicall link and a subcommand
```

Comment:
> applet

So not too related to these changes, but this is reminding me: should we link out to `App::subcommand_value_name` and `App::subcommand_help_heading`?

If you aren't familiar with what I'm referring to, see https://github.com/clap-rs/clap/pull/3078/files#diff-1b8c7992d48ee7a4f50a2f57b82a633371b175e1a90b543a34d4652379ade1d0

Reply:
It can't hurt, but it's not hugely useful yet:

```
$ ln -s target/debug/examples/busybox asdf
$ ./asdf
error: Found argument 'asdf' which wasn't expected, or isn't valid in this context

USAGE:
     [APPLET]

For more information try --help
```
I don't have an answer for #2862 yet, though it saying APPLET is useful.

Reply:
Having mulled this over in context of your proposed in-lining of the busybox applet, I think it would be useful.